src/entities/note.cpp

```diff
@@ -1229,10 +1215,11 @@ bool Note::storeNoteTextFileToDisk() {
     }
 
     const bool fileExists = this->fileExists();
+    const QString newName = _name;
 
     // assign the tags to the new name if the name has changed
     if (oldName != newName) {
-        if (!noteFileWasRenamed && TrashItem::isLocalTrashEnabled()) {
```

Comment:
Why did you remove this check?

Reply:
Its weird but i dont remember making this change

Reply:
Now I'm even more scared! :laughing:

Reply:
You removed the whole block where the variable was declared.

Reply:
I think it happened due to the renaming. very specific things are moved.

Reply:
Renaming of variables? What did you use for refactoring that?

Reply:
qt create :eyes:

Reply:
Why would it move blocks of code around when you rename a variable? :grimacing:

Reply:
Maybe that git did that all. e.g. I did https://github.com/pbek/QOwnNotes/commit/48806768f3d4d31fcae4df8fb01c0d6f4a56b85c#diff-69560e71be6fc4483e10036d29839d66R1235, maybe your PR ripped that all out again